traits/trait_types.py

```diff
@@ -69,7 +69,6 @@
 
 # Constants
 
-MutableTypes = (list, dict)
```

Comment:
Please also remove `MutableTypes` from the type stubs.

Reply:
ok. Thanks for reminding me.